README.md

```diff
@@ -0,0 +1,73 @@
+
+
+# Ansible Network Collection for Common Code (netcommon)
+[![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/builds?project=ansible-collections%2Fansible.netcommon) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/ansible.netcommon)](https://codecov.io/gh/ansible-collections/ansible.netcommon)
+
+The Ansible ``ansible.netcommon`` collection includes common network content to help automate the management of network devices.
+This includes  connection plugins, such as ``httpapi`` and ``netconf``.
+
+## Included content
+
+This collection includes:
+- Common network modules, such as ``net_*``, ``cli_*``, ``netconf_*``, and ``restconf_*`` modules.
+- Common connection plugins, such ``httpapi`` and ``network_cli``.
+- ``ipadrr`` and ``network`` filters.
```

Comment:
There are more filters. Should all of them be lister here https://github.com/ansible-collections/ansible.netcommon/blob/master/plugins/filter/network.py#L520-L528

https://github.com/ansible-collections/ansible.netcommon/blob/master/plugins/filter/ipaddr.py#L1157-L1177

Reply:
We have some documentation on these on docs.ansible.com An example is:
{{ output | parse_cli('path/to/spec') }}

Would that need to change to:
{{ output | ansible.necommon.parse_cli('path/to/spec') }} ?